_haikus/cornfalkes.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: cornflakes
+author: centicus1000
+---
+
+Hungry for breakfast
+pour some milk in the big bowl
```

Comment:
Please add either two spaces `  ` or a `<br>` tag to force a break after this line.